test.sh

```diff
@@ -87,7 +87,7 @@ assert 5 "i=0;while(i<10) if(i==5) return i;else i = i+1;i;"
 assert 10 "i=0;while(i<10) while(i<10)i=i+1;i;"
 assert 1 "{1;}"
 assert 10 "i=0;{i=i+2;i=i*5;}"
```

Comment:
```suggestion
assert 12 "i=0;{i=i+2;i=i*5;} return i+2;"
```

こうすると`foo();`の結果が変わってテストが通らなくなります。